Wikipedia/Code/wikipedia-language-variants.json

```diff
@@ -127,6 +127,15 @@
             "languageCode": "ku"
         }
     ],
+    "shi" : [
+        {
+            "languageVariantCode": "shi-tfng",
+            "languageName": "ⵜⴰⵛⵍⵃⵉⵜ ",
+            "localName": "Tachelhit (Tifinagh/ⵜⴰⵛⵍⵃⵉⵜ)",
```

Comment:
This is minor, but in the app languages list, `ⵜⴰⵛⵍⵃⵉⵜ` is already displayed as the main title of the cell. I suggest changing `localName` to `Tachelhit (Tifinagh)` to remove the duplicate names.

![Screenshot 2023-06-14 at 4 16 50 PM](https://github.com/wikimedia/wikipedia-ios/assets/3620196/c7aff243-a1c7-410d-b861-152cdb5af2e7)